src/util.rs

```diff
@@ -61,8 +61,25 @@ fn exit_with_code(r: reqwest::StatusCode) {
 
 // todo : assume https as the default scheme
 // Or get rid of this.
-pub fn url_validation(url: &str) -> Result<Url> {
-    Url::parse(url).context(format!("URL args: \'{}\' is not valid", url))
+pub fn url_validation(url: &str) -> Url {
+    let temp_url = Url::parse(url);
+    let temp_url = match temp_url {
+        Ok(pass) => {
+            https(pass)
+        },
+        Err(_error) => {
+            Url::parse(&format!("https://{}",url)).unwrap()
```

Comment:
This will panic if someone put some invalid data as an input (i.e. not a domain name nor an URL).